src/lib/support/TestGroupData.h

```diff
@@ -29,32 +29,31 @@ constexpr uint16_t kMaxGroupKeysPerFabric = 8;
 static chip::TestPersistentStorageDelegate sDeviceStorage;
 static chip::Credentials::GroupDataProviderImpl sGroupsProvider(sDeviceStorage, kMaxGroupsPerFabric, kMaxGroupKeysPerFabric);
 
-static const chip::FabricIndex kFabric1 = 1;
-static const chip::GroupId kGroup1      = 0x0101;
-static const chip::GroupId kGroup2      = 0x0102;
-static const chip::KeysetId kKeySet1    = 0x01a1;
-static const chip::KeysetId kKeySet2    = 0x01a2;
+static const chip::GroupId kGroup1   = 0x0101;
+static const chip::GroupId kGroup2   = 0x0102;
+static const chip::KeysetId kKeySet1 = 0x01a1;
+static const chip::KeysetId kKeySet2 = 0x01a2;
 
 } // namespace
 
 namespace chip {
 
 namespace GroupTesting {
 
-CHIP_ERROR InitGroupData()
+CHIP_ERROR InitGroupData(chip::FabricIndex fabric_index, const ByteSpan & compressed_fabric_id)
 {
     ReturnErrorOnFailure(sGroupsProvider.Init());
```

Comment:
This function used to initialize only one fabric, so only one call was needed. Since we need to initialize multiple fabrics, I have split the function in two: InitProvider(), and InitData(), the former is called once, while the latter is called once per fabric.